sustainability/models/account_move_line.py

```diff
@@ -98,6 +119,14 @@ def _compute_carbon_is_date_locked(self):
     # --------------------------------------------
 
     @api.depends(
+        # Seller
```

Comment:
This is not optimized because it triggers recompute on all account.move.line when you change your seller_ids or the product. Same for company lock date.